### ISSUE TITLE
Icon=hardinfo

### ISSUE THREAD

**Issue by arcolinuxz** [snwh/paper-icon-theme#764]
### Icon Request

Please provide the following in your request:

- Application name - /usr/bin/hardinfo - /usr/share/applications/hardinfo.desktop
- Icon=`hardinfo`
- Insert/attach a copy of the original icon for reference
https://github.com/lpereira/hardinfo

cs-cat-hardware might be a good link to it

**Comment by Secret-chest** [comment]
@arcolinuxz Dude this is abandoned

**Comment by arcolinuxz** [comment]
I see a lot of commits
https://github.com/lpereira/hardinfo/commits/master

**Comment by Secret-chest** [comment]
> I see a lot of commits
> https://github.com/lpereira/hardinfo/commits/master

@arcolinuxz No, Paper icon theme is abandoned

**Comment by Secret-chest** [comment]
No one besides me will ever reply

**Comment by snwh** [comment]
Yeah this project is effectively in cold-storage, I haven't had the time or interest in a long while to work on this.
